requirements.txt

```diff
@@ -5,6 +5,7 @@ drf-spectacular==0.18.1
 psycopg2-binary==2.9.1
 Pillow==8.3.1
 djangorestframework-simplejwt==4.8.0
+PyJWT==2.1.0
```

Comment:
Why?

Reply:
I had an error with response and I googled that it is the fault of the PyJWT update so I added the old one and it worked